handout/architect/architect-guide-source-code-to-working-software.md

```diff
@@ -0,0 +1,128 @@
+# Angular Source Code To Working Software
+
+_Scope Range_
+
+```
+    * Out of Scope *
+                                                    ^
+             Hardware                               |
+              Kernel                                | Possible passing reference 
+             Userland                               |
+            ----------                              v
+    JavaScript Runtime and/or DOM                   ^
+             ES5 Code                               |
+                                     TypeScript     | Layers referenced in 
+                                       Angular      | outline 
+                                       App Code     |
+                                                    v
+    * In Scope *
+    
+```
+
+## Act I - The Scene
+
+### Introduction
+
+- Describe the stack at an abstract level.  
+    - Possibly take a top down approach noting one aspect of the translation
+      moving down to the next layer
+    - Possibly take a bottom up approach while noting one thing that created a 
+      need for a new layer of abstraction 
+- Note that we're working with source code that is compiled, ie our code is not
+literally passed to the runtime "word for word"
+- Describe at an abstract level the complete process of turning our app code 
+into at a high level and how our commands work their way through the stack
+
+This would ideally take about five minutes as the audience should know this
+already
+
+
+### The Towering Web and Other Stacks
+
+_or why are we even using this stuff?_
+
+- Note the obvious fact that we could just write something like C but also note
+that it's much harder to share (provide) software to people or machines with
+traditional "Native Code".
+- Note that the Java world has "solved" this portability problem another way but
+also note that Java does not solve the _distribution_ aspect of the problem
```

Comment:
Don't understand which aspect of distribution you think JS has solved that J hasn't ?